src/components/Scorecards/ScorecardsServicePage/ScorecardsServicePage.tsx

```diff
@@ -117,10 +113,10 @@ export const ScorecardsServicePage = () => {
         </Box>
         <Box alignSelf="center">
           <Link
-            to={cortexScorecardServicePageURL({
+            to={cortexScorecardServicePageUrl({
               scorecardId,
               serviceId: score.serviceId,
-              cortexURL: cortexBaseUrl,
+              cortexUrl: cortexBaseUrl,
```

Comment:
we use URL in a lot of places but at least changed the new usages to use Url since I think we prefer that based on the FE capitalization thing we discussed a while ago

Reply:
Yeahhhh the inconsistency is going to drive people wild but 🤷‍♂️

Reply:
Well were already inconsistent so it's only moving us gradually towards the other direction 😃